cirq/ionq/ionq_client_test.py

```diff
@@ -317,3 +317,38 @@ def test_ionq_client_get_job_retry(mock_get):
                                           default_target='simulator')
     _ = client.get_job('job_id')
     assert mock_get.call_count == 2
+
+
+@mock.patch('requests.put')
+def test_ionq_client_cancel_job(mock_put):
```

Comment:
Should this and `delete_job` also have failure tests, or is the coverage provided by the `get` and `create` tests sufficient?

Reply:
Probably but better form to test these.  I've added tests.  What's the new acronym for the opposite of dry in tests?

Reply:
Internet tells me it's WET (for "Write Everything Twice") 😝 

I have a feeling these could be parameterized to DRY things up, but I'm not going to block the merge on that. LGTM